js/data/create_bucket.js

```diff
@@ -15,6 +16,10 @@ function createBucket(layer, buffers, collision, indices) {
         return;
     }
 
+    for (var k in layer.layout) {
+        layer.layout[k] = new StyleDeclaration('layout', layer.type, k, layer.layout[k]).value;
```

Comment:
Ah I see. Nope, worker_tile only ever knows about max z15 — so should I be threading the map's zoom through into the worker?